vector/src/main/java/im/vector/app/features/home/room/detail/RoomDetailViewModel.kt

```diff
@@ -550,7 +551,7 @@ class RoomDetailViewModel @AssistedInject constructor(
     private fun stopTrackingUnreadMessages() {
         if (trackUnreadMessages.getAndSet(false)) {
             mostRecentDisplayedEvent?.root?.eventId?.also {
-                viewModelScope.launch {
+                viewModelScope.launch(NonCancellable) {
```

Comment:
The (quite) new way to fix that properly is to use the session coroutine scope, like that:
`session.coroutineScope.launch {`

Reply:
Works, thanks!

Reply:
Thanks to you!